### DISCUSSION TITLE
[4.x]: Response of `/actions/users/login` should return user

### DISCUSSION THREAD

**Discussion by freder** [craftcms/cms#13374] [Ideas] [answered]
### Steps to reproduce
1. make a `POST` request to `/actions/users/login`

### Expected behavior
should return the user object as part of the response, if successful

### Actual behavior
response only contains `csrfTokenValue` and `returnUrl`

### Craft CMS version

4.4.14

### Installed plugins and versions

-

**Comment by brandonkelly** [comment, marked as answer]
Added for Craft 4.5 (820e8beac68a8cd265bddaaa5751f9a3f3631c36).

4.5 is still in development, but you can switch to the `4.5` branch by changing your `craftcms/cms` requirement in composer.json to `"4.5.x-dev as 4.5.0-alpha"`, and running `composer update`.

**Reply by brandonkelly** [reply]
Craft 4.5.0 is out now with that change.
